index.js

```diff
@@ -660,6 +722,8 @@ class PlayerState {
     this.isCollidedVertically = bot.entity.isCollidedVertically
     this.jumpTicks = bot.jumpTicks
     this.jumpQueued = bot.jumpQueued
+    // Hardcoded AABB because can't access getPlayerBB()
```

Comment:
It would be better to expose getPlayerBB()

Reply:
Should I shove `physics` into a json and move `getPlayerBB(pos)` outside of the physics class?